src/components/underground.html

```diff
@@ -45,28 +49,28 @@
                     <div class='row no-gutters'>
                         <button class='col-12 col-md-4 btn btn-primary' onClick='Mine.toolSelected(Mine.Tool.Hammer)'>Hammer (<knockout data-bind="text: Underground.HAMMER_ENERGY"></knockout> energy)</button>
                         <button class='col-12 col-md-4 btn btn-info' onClick='Mine.toolSelected(Mine.Tool.Chisel)'>Chisel (<knockout data-bind="text: Underground.CHISEL_ENERGY"></knockout> energy)</button>
-                        <button
-                            class="col-12 col-md-4 btn btn-secondary"
-                            id="#mine-prospect-result"
-                            data-bind="
-                                text: Mine.itemsFound() + ' / ' + Mine.itemsBuried() + ' items found',
+                        <button class="col-12 col-md-4 btn btn-secondary"
+                                id="#mine-prospect-result"
+                                data-bind="
+                                text: Mine.itemsFound() + ' / ' + Mine.itemsBuried() + ' items found' ,
                                 tooltip: {
-                                    title: () => Mine.prospectResult() || 'Prospect to get more details',
-                                    trigger: 'hover',
-                                    placement:'top',
-                                    html: true,
-                                }
+                                title: ()=>
+                            Mine.prospectResult() || 'Use a Survey Charge to get more details',
+                            trigger: 'hover',
+                            placement:'top',
+                            html: true,
+                            }
                             "
-                        >
+                            >
+                        </button>
+                        <button class='col-12 col-md-4 btn btn-success'
+                                onClick='Mine.prospect()'
+                                data-bind="disable: Mine.prospectResult() ? true : false">
+                            Survey Charge (<knockout data-bind="text: App.game.underground.getSurveyCharge_Cost()"></knockout> energy)
```

Comment:
Can we keep this as `Survey` or `Prospect` as it causes wrapping on smaller screens
```suggestion
                            Survey (<knockout data-bind="text: App.game.underground.getSurveyCost()"></knockout> energy)
```

Reply:
Done. Changed to Survey.